src/foam/u2/grid/seutil.js

```diff
@@ -0,0 +1,396 @@
+console.log("Loading SE.util");
```

Comment:
foam.CLASS() defines classes, which need to be instantiate to create instances before their methods can be used, but foam.LIB() just creates a stack collection of functions, like you're doing here.  If you look at src/foam/core/stdlib.js you'll see lots of uses (and some useful lib functions).